src/hooks.ts

```diff
@@ -0,0 +1 @@
+export { useConfig } from './app/hooks/useConfig.js'
```

Comment:
Will probably need an entrypoint in `package.json#exports` for this.

Reply:
Done